Promact.Oauth.Server/src/Promact.Oauth.Server/Startup.cs

```diff
@@ -60,9 +57,23 @@ public Startup(IHostingEnvironment env, ILoggerFactory loggerFactory)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            //var builder = new ContainerBuilder();
-            //builder.RegisterType<AuthMessageSender>().As<IEmailSender>().InstancePerDependency();
-            //builder.RegisterType<AuthMessageSender>().As<ISmsSender>().InstancePerDependency();
+            // Configure AppSettings using config by installing Microsoft.Extensions.Options.ConfigurationExtensions
+            //services.Configure<AppSettings>(Configuration);
+
+            //Configure AppSettings using config by installing Microsoft.Extensions.Options.ConfigurationExtensions
+            services.Configure<AppSettings>(myOptions =>
```

Comment:
What is the point of keeping appsettings section separately when you combine everything again in code? If sections are different in appsettings.json then in code too. It should be different. You are not taking benefit of aspnet core's capabilities.